### PR TITLE
chore: refactor gauge chart entity

### PR DESCRIPTION
**Pull request by JeevaRamu0104**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description

Enhance performance entity to accept dynamic argument while creating the chart data.
Currently payment failure is not supporting the entity structure.
image
 

## How did you test it?
We can check the payment failure chart there shouldn't be any changes.

## Where to test it?
- [x] INTEG
- [x] SANDBOX
- [ ] PROD

## Checklist

- [ ] I ran `npm run re:build`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: src/screens/Analytics/PerformanceMonitor/PerformanceMonitorEntity.res [hunk 1]
```diff
@@ -239,9 +254,12 @@ let tableEntity = EntityType.makeEntity(
   ~getHeading,
 )
 
-let getFailureEntity: entity<array<errorObject>> = {
+let getFailureEntity: entity<array<errorObject>, option<string>> = {
   getChartOption: _ => Dict.make()->JSON.Encode.object,
-  getChartData: (~array as _, ~config as _) => [],
+  getChartData: (~args) => {
+    let _ = args
```
Comment (PritishBudhiraja) [inline src/screens/Analytics/PerformanceMonitor/PerformanceMonitorEntity.res:260]: why 
let _ = args?
can we directly return []

Comment (JeevaRamu0104) [reply src/screens/Analytics/PerformanceMonitor/PerformanceMonitorEntity.res:260]: because its was giving a warning of unused variables

Comment (PritishBudhiraja) [reply src/screens/Analytics/PerformanceMonitor/PerformanceMonitorEntity.res:260]: it can be written as
```
  getChartData: (~args as _) => [],
```

**Review by PritishBudhiraja — APPROVED** [review]
LGTM Code wise :)

**Review by sagarnaikjuspay — APPROVED** [review]

**Pull request merged**